packages/common/src/extensions/slickCellExternalCopyManager.ts

```diff
@@ -280,11 +280,7 @@ export class SlickCellExternalCopyManager {
 
     // ignore new rows if we don't have a "newRowCreator"
     if ((availableRows < destH) && typeof this._addonOptions.newRowCreator === 'function') {
-      const d = this._dataWrapper.getDataItems();
-      for (addRows = 1; addRows <= (destH - availableRows); addRows++) {
-        d.push({});
-      }
-      this._dataWrapper.setDataItems(d);
```

Comment:
but if you remove this, then how will the item be added to the dataset? You're forcing the user to do that part itself, is that it? isn't this a breaking change though?

Reply:
it is but the old behavior didnt work either as it clashed with existing ids, more so it actually created elements with an undefined id

Reply:
can we somehow add some console warning in the case that the user forget to add it?

Reply:
@zewa666 I've opened my PR for the Vitest migration and since it's touching every test files, it will certainly conflict with your PR. So I'd like to merge your PR before Vitest but would it be possible to add a console warning like I mentioned just above?

Reply:
sure will try to do so today

Reply:
took a bit longer but now it's in. Was it something like this you meant?

Reply:
The console warning I asked for was mainly for what you wrote above, if the user forgets to do it himself, then show them a warning that they didn't. I'm not sure if your new console covers this or not, you know more than I do on that part. 

> and yes, I'd expect the user to do it by himself as thats actually the only safe way to handle it imo

Side note, this new console will be missing test coverage, you could also ignore it with Istanbul (or v8 for upcoming Vitest), though if you can add a unit test then use something like below for spying on the console

https://github.com/ghiscoding/slickgrid-universal/blob/e97017aa786259fb947cff39f1ce4204c0cfedee/packages/utils/src/__tests__/domUtils.spec.ts#L90-L93

Reply:
I guess thats the only way to check that, cant make any assumptions about how the resulting ids should look like.

yep will update the tests. thx for the sample console check